### PR TITLE
Parse email files- signd email fix

### PR DESCRIPTION
**Pull request by reutshal**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/20193

## Description
Added support in another version of smime eml.

## Does it break backward compatibility?
   - No

## Must have
- [X] Tests
- [ ] Documentation (with link to it)
- [X] Code Review

## Technical writer review
Mention and link to the files that require a technical writer review.
- [] [YAML file](link)
- [X] [CHANGELOG](https://github.com/demisto/content/blob/parseEmailFiles-signd_email_fix/Scripts/ParseEmailFiles/CHANGELOG.md)

### CODE REVIEW

File: Scripts/ParseEmailFiles/ParseEmailFiles.py [hunk 1]
```diff
@@ -43,6 +43,7 @@
 sys.setdefaultencoding('utf8')  # pylint: disable=no-member
 
 MAX_DEPTH_CONST = 3
+GOT_INTO_REC = False
```
Comment (ronykoz) [inline Scripts/ParseEmailFiles/ParseEmailFiles.py]: Not a good name, please update that

File: Scripts/ReadPDFFileV2/ReadPDFFileV2.py [hunk 2]
```diff
@@ -282,3 +282,145 @@ def main():
 # python2 uses __builtin__ python3 uses builtins
 if __name__ == "__builtin__" or __name__ == "builtins":
     main()
+
+
+
+
+
+
+  import pdfx
```
Comment (glicht) [inline Scripts/ReadPDFFileV2/ReadPDFFileV2.py]: Why are you changing this file?

Comment (reutshal) [reply Scripts/ReadPDFFileV2/ReadPDFFileV2.py]: Reverting

Comment (glicht) [reply Scripts/ReadPDFFileV2/ReadPDFFileV2.py]: the change is still there

**Pull request merged**